py_zipkin/zipkin.py

```diff
@@ -163,6 +164,10 @@ def __init__(
         :param host: Contains the ipv4 value of the host. The ipv4 value isn't
             automatically determined in a docker environment
         :type host: string
+        # CK45
```

Comment:
What's this comment?

Reply:
It was kind of a bookmark for me, forgot to remove it.